### ISSUE TITLE
Windows are not blurred when I use a transparent color scheme

### ISSUE THREAD

**Issue by pnedyalkov91** [paulmcauley/classik#34]
Description: When I use a transparent color scheme, windows are not blurred. I'm using "Force Blur" but nothing happens.
Here is my color scheme:
https://pastebin.com/UK4Yhhfb

Configuration:
Operating System: Arch Linux
KDE Plasma Version: 5.25.0
KDE Frameworks Version: 5.95.0
Qt Version: 5.15.4
Kernel Version: 5.18.4-arch1-1 (64-bit)
Graphics Platform: X11
Processors: 12 × Intel® Core™ i7-9750H CPU @ 2.60GHz
Memory: 15.5 GiB of RAM
Graphics Processor: NVIDIA GeForce RTX 2060/PCIe/SSE2
Manufacturer: LENOVO
Product Name: 81Q4
System Version: LENOVO LEGION Y540-17IRH

![image](https://user-images.githubusercontent.com/59796383/174256211-2ff369b6-46dc-46da-be8c-c3de8e948266.png)

**Comment by paulmcauley** [comment]
What is "Force Blur"?

In order to prevent the kornerbug I have to specify a region to blur which is just the header area, and is blurring correctly. My hunch here is that "Force Blur" doesn't specify a region (though I don't know for sure).

Does it work if under ClassiK->Titlebar you untick "Apply these opacity settings to entire Header area"?

**Comment by pnedyalkov91** [comment]
Hello Paul,

I would like to thank you for your work! Your theme is awesome!

Force blur is an addon for Kwin. You can find more information at:
https://github.com/esjeon/kwin-forceblur

I unticked the setting but the background window turned into black:
![image](https://user-images.githubusercontent.com/59796383/174265873-7426ecc4-4f61-427c-bd00-368590794bc8.png)

**Comment by paulmcauley** [comment]
Does "Force Blur" work for you with the Breeze default window decoration and application style? For me, in Breeze it turns black as well.

**Comment by pnedyalkov91** [comment]
No, it doesn't. It only works with the "Lightly" application style. Here is a screenshot with "classik" windows decoration and "lightly" application style:
https://i.imgur.com/gql578v.png

**Comment by paulmcauley** [comment]
That would explain it. I have forked from Breeze. I would have to investigate what Lightly and Force Blur are doing.

**Comment by pnedyalkov91** [comment]
Thank you!

**Comment by pnedyalkov91** [comment]
Hello Paul,
Yes, it was taken on Plasma 5.25. Have you already imported and applied my color scheme (https://pastebin.com/UK4Yhhfb)?
If yes, you can restart X Server and try again. Here is my "Force Blur" configuration:
![image](https://user-images.githubusercontent.com/59796383/174308082-0c37c6e5-f63a-41b1-aaa1-4b44bbc2eb08.png)

**Comment by paulmcauley** [comment]
Hi, I can now get ForceBlur to (very partially) work with _Force Blur_ + _Lightly_ application style + _ClassiK_ window decoration + your _Shaker Dark_ colours. I was affected by the following Lightly bug so had to test on a different monitor:
https://github.com/Luwx/Lightly/issues/180

This is it when it looks good:
![classik_lightly](https://user-images.githubusercontent.com/6924479/174649671-4c40e91a-908c-4bd9-8373-b8c6313e1392.png)

However, there are too many cases where  _Force Blur_ + _Lightly_ don't work either -- too many for them to be usable. For example:
![classik+lightly_bugged](https://user-images.githubusercontent.com/6924479/174650438-3398fc4b-c439-4d70-9d06-2e000294f746.png)


I will try to tweak the transparency over time with ClassiK to see if I can get something equivalent or better than Lightly. Looking at Force Blur won't be a high priority though, as it doesn't work all the time even with your Lightly example, and is X11 only.
